Packs/Netcraft_V2/Integrations/Netcraft/Netcraft.yml

```diff
@@ -0,0 +1,1501 @@
+    - description: A comma separated list of XSOAR file entry IDs to report. Max 10 files per submission, with a maximum combined file size of 20MiB. The arguments "file_content" and "file_name" can be used for reporting a single file.
+      isArray: true
+      name: entry_id
+    - description: The reason the file is considered malicious, should be less than 1000 characters.
+      name: reason
+    - auto: PREDEFINED
+      description: Use Cortex XSOAR built-in polling to wait for the report to be processed.
+      name: polling
+      defaultValue: 'true'
+      predefined:
+      - 'true'
+      - 'false'
+    - defaultValue: '30'
+      description: Indicates how long to wait between command executions (in seconds) when the 'polling' argument is true. Minimum value is 10 seconds.
+      name: interval_in_seconds
+    - defaultValue: '600'
+      description: Indicates the time in seconds until the polling sequence times out.
+      name: timeout
+    outputs:
+    - contextPath: Netcraft.Submission.date
+      description: The unix timestamp of the submission.
+      type: Number
+    - contextPath: Netcraft.Submission.source_name
+      description: The name of the source of this submission.
+      type: String
+    - contextPath: Netcraft.Submission.state
+      description: The state of the submission. One of "processing", "no threats", "suspicious" or "malicious". Note, in the case of a misclassification, a submission may be assigned a higher-severity state several days after its initial classification.
+      type: String
+    - contextPath: Netcraft.Submission.submitter_email
+      description: The email address of the reporter of the submission.
+      type: String
+    - contextPath: Netcraft.Submission.uuid
+      description: The unique identifier of the submission.
+      type: String
+    - contextPath: Netcraft.Submission.classification_log.date
+      description: A unix timestamp of when this state change occurred.
+      type: Number
+    - contextPath: Netcraft.Submission.classification_log.from_state
+      description: The state the submission moved out of.
+      type: String
+    - contextPath: Netcraft.Submission.classification_log.to_state
+      description: The state the submission moved into.
+      type: String
+    - contextPath: Netcraft.Submission.files
+      description: An API URL to get details about the files associated with this submission.
+      type: String
+    - contextPath: Netcraft.Submission.has_cryptocurrency_addresses
+      description: Whether the submission contains cryptocurrency addresses.
+      type: Boolean
+    - contextPath: Netcraft.Submission.has_files
+      description: Whether the submission contains files.
+      type: Boolean
+    - contextPath: Netcraft.Submission.has_issues
+      description: Whether the submission contains issues.
+      type: Boolean
+    - contextPath: Netcraft.Submission.has_mail
+      description: Whether the submission has a mail.
```

Comment:
Do you mean email address?
If so, should be: Whether the submission has an email address.

Reply:
It means a literal email.